tensorflow/tools/api/golden/v1/tensorflow.train.-looper-thread.pbtxt

```diff
@@ -36,7 +36,7 @@ tf_class {
   }
   member_method {
     name: "join"
-    argspec: "args=[\'self\', \'timeout\'], varargs=None, keywords=None, defaults=[\'None\'], "
+    argspec: "args=[\'self\', \'timeout\', \'balancing\'], varargs=None, keywords=None, defaults=[\'None\', \'True\'], "
```

Comment:
That does look suspicious as well

Reply:
This was suggested by the api compatibility test in order for the test to pass. I will remove this.